ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManagerTestBuilder.java

```diff
@@ -208,7 +208,7 @@ public EthProtocolManager build() {
               25,
               25,
               false,
-              SyncMode.FAST,
```

Comment:
as above, any test classes can be updated in advance of removing the feature itself